docs/main_functionalities.rst

```diff
@@ -0,0 +1,40 @@
+####################
+Main functionalities
```

Comment:
```suggestion
Main Features
```
Suggest replacing "functionalities" with "features" everywhere, including renaming this file